Makefile.PL

```diff
@@ -27,6 +27,7 @@ my %WriteMakefileArgs = (
     "DateTime::HiRes" => 0,
     "Digest::MD5" => 0,
     "Exporter" => 0,
+    "Feature::Compat::Try" => 0,
```

Comment:
So this commits us to a move to 5.14 as base perl - not sure I am against that but I think we should probably plan/announce a depreciation plan before doing this

Reply:
I'm not opposed to bumping the requirements to a 5.10 or higher. Perl 5.8 is OLD, 20 years old. 5.10 and up allows us to use more modern syntax.

You could also use Try::Tiny, which would keep the 5.8 requirement.